template.yaml

```diff
@@ -2,11 +2,25 @@ kind: Template
 apiVersion: v1
 template: spark
 metadata:
-  name: spark
+  name: apache-spark
+  creationTimestamp: null
+  annotations:
+    description: Apache Spark with 3 worker nodes.
+    iconClass: icon-nodejs
+    tags: quickstart,Apache,Spark,apachespark
 labels:
   app: sparkcluster
 objects:
-
+- apiVersion: v1
+  kind: PersistentVolumeClaim
```

Comment:
hehe, oops. +1 on both accounts @tmckay!

fyi, these are templates for kubernetes based deployments and we do have a few corner cases where persistent volumes *might* be useful, but i tend to agree in the general sense that spark users will want some sort of cloud based storage like HDFS or other things (eg databases, s3, etc).